### DISCUSSION TITLE
customSetup autocomplete

### DISCUSSION THREAD

**Discussion by Vu-Ngoc-Trung** [codesandbox/sandpack#1148] [Q&A]
how to setup autocomplete with lib type avilable with sanpack, i use customSetup but not working

**Comment by danilowoz** [comment]
Did you see this section inside the documentation? If so, can you provide more detail it didn't work? 
https://sandpack.codesandbox.io/docs/advanced-usage/components#extensions

Thanks

**Reply by Vu-Ngoc-Trung** [reply]
I want my own custom type in customsetup, i want autocomplete use type lib with customsetup
